ui/webui/resources/BUILD.gn

```diff
@@ -234,6 +234,7 @@ leo_icons = [
   "ecosia-color.svg",
   "edit-box.svg",
   "edit-pencil.svg",
+  "email-shield.svg",
```

Comment:
@petemill is this right that all of these resources go in one file and don't get split up for each page?

Reply:
These icons are available from `://resources/brave-icons` and are available across all pages. Many of these are shared across multiple WebUIs and across Polymer, Lit and React UIs.

It's roughly our equivalent of `ui/webui/resources/cr_elements/icons.html.ts` except pages only need to load the icons they actually use.